samples/nrf9160/modem_shell/src/fota/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_include_directories(.)
```

Comment:
Why does Zephyr need to know the app includes ?

Reply:
Fixed